Xamarin port

AdiMarianMutu opened the pull request.

AdiMarianMutu committed:
Update README.md

AdiMarianMutu committed:
Create OEMBatteryWhitelist.cs

AdiMarianMutu committed:
Update README.md

AdiMarianMutu committed:
Update OEMBatteryWhitelist.cs

Xxshark888xX commented:
@judemanutd Let me know if you want to change the class name or anything else.

Thanks to you I was able to make my app working on more devices while the phone is in sleep mode, thus you deserve a big thank you! 😁

judemanutd commented:
Hey @Xxshark888xX thanks but the credit doesn't entirely go to me either, this library is part my effort but also sources alot of the information from stackoverflow and various other github issues and repositories that I came across. 

I am a bit busy during the weekdays but should be able to review the PR over the weekend and get back to you, thanks for the PR.

AdiMarianMutu committed:
Fixed bug with brand name to lowercase

As reported in this comment, there's no need to use local culture characters when lowercase the brand name

https://github.com/judemanutd/AutoStarter/issues/48

AdiMarianMutu committed:
Fixed bug with brand name to lowercase

As reported in this comment, there's no need to use local culture characters when lowercase the brand name

https://github.com/judemanutd/AutoStarter/issues/48
